Make Purple Yolk work on Windows

EduardSergeev opened the pull request:
Correct path format in `cwd` argument

Use [Uri.fsPath](https://code.visualstudio.com/api/references/vscode-api#Uri.fsPath) (instead of `path`) which returns URI file path in platform-specific format.

This version should work on all threee supported platforms: Linux, MacOS & Windows

Fixes: #85

tfausak reviewed (approved):
Looks great, thanks!

tfausak committed:
Merge branch 'main' into fix/windows

EduardSergeev committed:
Make extension work on Windows

Previously `childProcess.spawn` was failing on Windows due to invalid path format in `cwd` argument
Use `Uri.fsPath` (instead of `Uri.path`) which provides correct (platform specific) file paths
